### PR TITLE
feat: migrate from uglifyjs-webpack-plugin to terser-webpack-plugin

### PR DESCRIPTION
**Pull request by skarunakar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,7 +73,8 @@
     "react-motion": "0.5.2",
     "react-required-if": "^1.0.3",
     "react-resize-aware": "2.7.0",
-    "react-router": "3.2.0"
+    "react-router": "3.2.0",
+    "terser-webpack-plugin": "^4.2.3"
```
Comment (dholbach) [inline package.json:77]: I guess this could be under `devDependencies` as well - just like `uglifyjs-webpack-plugin` before?

**Comment by dholbach** [comment]
Thanks a lot for your work on this!

File: package.json (hunk 1 above)
Comment (skarunakar) [reply package.json]: Agreed

File: package.json [hunk 2]
```diff
@@ -139,7 +139,7 @@
     "stylelint-declaration-use-variable": "1.7.2",
     "stylelint-processor-styled-components": "1.3.1",
     "superagent": "^3.6.0",
-    "uglifyjs-webpack-plugin": "^2.2.0",
+    "terser-webpack-plugin": "^4.2.3",
```
Comment (skarunakar) [inline package.json:142]: We cannot use latest terser-webpack-plugin here as it is supported only with webpack v5.
This repo uses v4

**Comment by jpellizzari** [comment]
@skarunakar Can you give some more context on why you created this pull request?

**Comment by jpellizzari** [comment]
> @skarunakar Can you give some more context on why you created this pull request?

Nevermind, I see the linked issue.

**Review by jpellizzari — APPROVED** [review]

**Pull request merged**